sdk/servicebus/service-bus/test/public/atomManagement.spec.ts

```diff
@@ -1338,701 +1191,491 @@ describe("Atom management - Authentication", function(): void {
-      forwardTo: managementQueue1
-    },
-    output: {
-      lockDuration: "PT1M",
-      maxDeliveryCount: 10,
-      defaultMessageTimeToLive: "P10675199DT2H48M5.4775807S",
-      deadLetteringOnFilterEvaluationExceptions: true,
-      deadLetteringOnMessageExpiration: false,
-      enableBatchedOperations: true,
-      requiresSession: false,
-
-      forwardDeadLetteredMessagesTo: `${endpointWithProtocol}${managementQueue1}`,
-      forwardTo: `${endpointWithProtocol}${managementQueue1}`,
-      autoDeleteOnIdle: "P10675199DT2H48M5.4775807S",
-
-      defaultRuleDescription: undefined,
-
-      messageCount: 0,
-      enablePartitioning: undefined,
-      enableExpress: undefined,
-      maxSizeInMegabytes: undefined,
-      sizeInBytes: undefined,
-
-      userMetadata: undefined,
-      messageCountDetails: undefined,
-      status: "Active",
-      availabilityStatus: "Available",
-      subscriptionName: managementSubscription1,
-      topicName: managementTopic1
-    }
-  },
-  {
-    testCaseTitle: "pass in absolute URI for forwardTo and forwardDeadLetteredMessagesTo",
-    input: {
-      forwardDeadLetteredMessagesTo: `${endpointWithProtocol}${managementQueue1}`.toUpperCase(),
-      forwardTo: `${endpointWithProtocol}${managementQueue1}`.toUpperCase()
-    },
-    output: {
-      forwardDeadLetteredMessagesTo: `${endpointWithProtocol}${managementQueue1.toUpperCase()}`,
-      forwardTo: `${endpointWithProtocol}${managementQueue1.toUpperCase()}`
-    }
-  }
-].forEach((testCase) => {
-  describe(`createSubscription() using different variations to message forwarding related parameters in "subscriptionOptions"`, function(): void {
-    beforeEach(async () => {
-      await recreateQueue(managementQueue1);
-      await recreateTopic(managementTopic1);
-    });
-
-    afterEach(async () => {
-      await deleteEntity(EntityType.TOPIC, managementTopic1);
-      await deleteEntity(EntityType.QUEUE, managementQueue1);
-    });
-
-    it(`${testCase.testCaseTitle}`, async () => {
-      const response = await createEntity(
-        EntityType.SUBSCRIPTION,
-        managementSubscription1,
-        managementTopic1,
-        undefined,
-        true,
-        undefined,
-        undefined,
-        testCase.input
-      );
 
-      should.equal(
-        response.subscriptionName,
-        managementSubscription1,
-        "Subscription name mismatch"
-      );
-      should.equal(response.forwardTo, testCase.output.forwardTo, "forwardTo value mismatch");
-      should.equal(
-        response.forwardDeadLetteredMessagesTo,
-        testCase.output.forwardDeadLetteredMessagesTo,
-        "forwardDeadLetteredMessagesTo value mismatch"
-      );
+        should.equal(response.name, managementTopic1, "Topic name mismatch");
+        assert.deepEqualExcluding(response, testCase.output, [
+          "_response",
+          "createdAt",
+          "modifiedAt",
+          "accessedAt",
+          "maxMessageSizeInKilobytes"
```

Comment:
Though not configurable, looks like the "Standard" namespaces have this set.
And among Standard namespaces, maxMessageSizeInKilobytes is `256` for non-Canary namespace and `1024` for a Canary namespace.

Hence, excluding the property from the assertion.